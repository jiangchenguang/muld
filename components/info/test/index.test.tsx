import * as React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render, cleanup } from '@testing-library/react';
import Info from '..';

describe('Icon', () => {
    afterEach(cleanup);

    it('prop', () => {
        const { asFragment } = render(<Info badge="40" />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('prop', () => {
        const { asFragment } = render(<Info badge="" />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('prop', () => {
        const { asFragment } = render(<Info dot />);
        expect(asFragment()).toMatchSnapshot();
    });
});
