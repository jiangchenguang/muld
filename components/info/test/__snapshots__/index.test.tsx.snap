// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon prop 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm bkXGDZ mul-info"
  >
    40
  </div>
</DocumentFragment>
`;

exports[`Icon prop 2`] = `<DocumentFragment />`;

exports[`Icon prop 3`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm bkXGDZ mul-info mul-info--dot"
  />
</DocumentFragment>
`;
